### PR TITLE
developer-friendly script for running integration tests locally

### PR DESCRIPTION
**Pull request by cellog**
# Description

This PR introduces a new script, which is designed to make it easy to run the integration tests locally in a docker context repeatedly. It starts by using `docker down` to remove any running instances of the unlock integration tests, mainly `unlock`, `ganache-integration`. This also removes networks (a critical step). Next, it builds (or re-builds) the docker images locally using local cache, and finally it runs the integration tests

# Issues

Refs #1890 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Please add nore info inside the file itself about its purpose and how to use it ;)

**Comment by cellog** [comment]
> Please add nore info inside the file itself about its purpose and how to use it ;)

done, PTAL

File: scripts/local-docker-integration-tests.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export DB_HOSTNAME='db'
+export CI=true
+export HTTP_PROVIDER='ganache-integration'
+export LOCKSMITH_URI='http://locksmith:8080'
+export PAYWALL_URL=http://unlock:3000/paywall
+export PAYWALL_SCRIPT_URL=http://unlock:3000/static/paywall.min.js
+
+# if the integration test images are running, this ensures we remove any state
+# prior to attempting to run the tests. This line is critical!
+docker-compose -f $DOCKER_COMPOSE_FILE down --volumes
```
Comment (akeem) [inline scripts/local-docker-integration-tests.sh]: Is `volumes` needed here?

Comment (cellog) [reply scripts/local-docker-integration-tests.sh]: I guess not, we don't define any do we? I'll pull that out

**Comment by cellog** [comment]
> I worry about the duplicate information between this file and `integration-tests.sh`.
> Could we only have the strictly necessary stuff in here and then invoke integration-tests.sh as the last step?

sure, I'll look into that, good idea!

**Comment by cellog** [comment]
ok @julien51 PTAL.

We ended up needing the environment variables in both scripts, so I extracted them out. The odd syntax:

```bash
. $REPO_ROOT/scripts/integration-envs.sh
```

is needed to source the env variables. It's not at all intuitive, so I added a comment above to help clarify.

**Comment by cellog** [comment]
update: this approach does not work. I don't think it's worth the effort to remove duplication, but I will have it invoke the integration_tests.sh script

**Pull request merged**